recipe/meta.yaml

```diff
@@ -30,10 +30,14 @@ requirements:
     - setuptools_scm
     - numpy 1.16
     - pandas
+    - arrow-cpp
+    - pyarrow
   run:
     - python
     - {{ pin_compatible('numpy') }}
     - pandas
+    - arrow-cpp
+    - pyarrow
```

Comment:
This should not be pinned as strictly as the run_exports as it is only using the C-API.

Reply:
so how strict do we want it?

Reply:
seems like arrow-ccp is still x.x.x ? should this just be x?

Reply:
`arrow-cpp >=1` sounds reasonable for me.

Reply:
I would still use `x.x.x` for the C++ ABI, there no tests exist for ABI stability and nobody takes care of it. The C-API is very stable but also extremely limited.

Reply:
This needs a `ignore_run_exports: - arrow-cpp`. Otherwise the runtime will still be pinned.